### ISSUE TITLE
Not able to use TextInput

### ISSUE THREAD

**Issue by ponchautf** [software-mansion/radon-ide#333]
**Description*
When I try to use TextInput in the IDE ios simulator, it's not possible to enter text.
I only see a menu that invite to fill the field
<img width="457" alt="Capture d’écran 2024-05-31 à 10 29 00" src="https://github.com/software-mansion/react-native-ide/assets/28573898/212f25a9-b4ac-4ddc-a2cb-98974bb8c02c">

(not yet tried on android simulator
**Environment**
React native IDE 0.0.14
React Native 0.73.6
VS Code 1.89.1
MacBook Pro 2019 (Intel core i9)

**Comment by kmagiera** [comment]
Thanks for reporting, does it behave like that on all text input fields or some specific ones? Can you reproduce this on some empty projects too? Can you share a reproducer repo?

**Comment by ponchautf** [comment]
Hi, 
Unfortunately, I can't share the repo, it's a private one.
Textnput component came from react-native (it's not a custom one) and used in a prety simple component:

````
<>
<Text style={typography.BodyRegular}>
Si vous avez des questions, des commentaires ou des préoccupations concernant 
<Text style={typography.BodyBold}>Marches Points Verts</Text>,
n'hésitez pas à nous contacter en utilisant le formulaire ci-dessous. Nous sommes là pour vous aider et nous nous efforcerons de répondre à votredemande dans les plus brefs délais.
</Text>
<Spacer size={20} />
<Text style={typography.HeadlineRegular}>Nom complet : </Text>
<TextInput style={styles.input} value={name} onChangeText={(text) => setName(text)} />
<Text style={typography.HeadlineRegular}>Adresse e-mail : </Text>
<TextInput style={styles.input} value={email} onChangeText={(text) => setEmail(text)} />
<CustomButton onPress={sendForm} text="Envoyer" disabled={!isChecked} />
</>
````

I got the same behavior on all TextInput.
I can't try on android, react-native IDE won't allow me to start an android emulator 
(I'm working on old intel MacBookPro and got an error CPU Architecture 'x64' is not supported by the QEMU2 emulator)

**Comment by ponchautf** [comment]
FYI : meantime, I found the workaround to start android simulator on intel laptop, but it's not working.
Emulator is starting but remain stuck in "Waiting for app to load"

I just see a metro circular reference error in output windows that does not occur with iOS simulator.

**Comment by ovidb** [comment]
We have the same issue on all types of inputs

![2024-07-31 11 01 48](https://github.com/user-attachments/assets/e6923cf1-6f4b-4583-9408-2171359d195d)

**Comment by kmagiera** [comment]
a lot has changed since this got repored. I tried to reproduce this now but with no success. Can anyone here try and confirm whether its happening on the most recent 0.21 version of the IDE?

**Comment by kmagiera** [comment]
We managed to reproduce this eventually and it is fixed as of #718
